runtime/polkadot/src/weights/xcm/pallet_xcm_benchmarks_fungible.rs

```diff
@@ -0,0 +1,139 @@
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-04-13, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("polkadot-dev"), DB CACHE: 1024
+
+// Executed Command:
+// target/production/polkadot
+// benchmark
+// pallet
+// --steps=50
+// --repeat=20
+// --extrinsic=*
+// --execution=wasm
+// --wasm-execution=compiled
+// --heap-pages=4096
+// --json-file=/var/lib/gitlab-runner/builds/zyw4fam_/0/parity/mirrors/polkadot/.git/.artifacts/bench.json
+// --pallet=pallet_xcm_benchmarks::fungible
+// --chain=polkadot-dev
+// --header=./file_header.txt
+// --template=./xcm/pallet-xcm-benchmarks/template.hbs
+// --output=./runtime/polkadot/src/weights/xcm/
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weights for `pallet_xcm_benchmarks::fungible`.
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo<T> {
+	// Storage: System Account (r:1 w:1)
+	// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: MaxEncodedLen)
+	pub(crate) fn withdraw_asset() -> Weight {
+		Weight::from_ref_time(23_068_000 as u64)
+			.saturating_add(T::DbWeight::get().reads(1 as u64))
+			.saturating_add(T::DbWeight::get().writes(1 as u64))
+	}
+	// Storage: System Account (r:2 w:2)
+	// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: MaxEncodedLen)
+	pub(crate) fn transfer_asset() -> Weight {
+		Weight::from_ref_time(47_403_000 as u64)
+			.saturating_add(T::DbWeight::get().reads(2 as u64))
+			.saturating_add(T::DbWeight::get().writes(2 as u64))
+	}
+	// Storage: System Account (r:2 w:2)
+	// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: MaxEncodedLen)
+	// Storage: Configuration ActiveConfig (r:1 w:0)
+	// Proof Skipped: Configuration ActiveConfig (max_values: Some(1), max_size: None, mode: Measured)
+	// Storage: XcmPallet SupportedVersion (r:1 w:0)
+	// Proof Skipped: XcmPallet SupportedVersion (max_values: None, max_size: None, mode: Measured)
+	// Storage: XcmPallet VersionDiscoveryQueue (r:1 w:1)
+	// Proof Skipped: XcmPallet VersionDiscoveryQueue (max_values: Some(1), max_size: None, mode: Measured)
+	// Storage: XcmPallet SafeXcmVersion (r:1 w:0)
+	// Proof Skipped: XcmPallet SafeXcmVersion (max_values: Some(1), max_size: None, mode: Measured)
+	// Storage: Dmp DownwardMessageQueues (r:1 w:1)
+	// Proof Skipped: Dmp DownwardMessageQueues (max_values: None, max_size: None, mode: Measured)
+	// Storage: Dmp DownwardMessageQueueHeads (r:1 w:1)
+	// Proof Skipped: Dmp DownwardMessageQueueHeads (max_values: None, max_size: None, mode: Measured)
+	pub(crate) fn transfer_reserve_asset() -> Weight {
+		Weight::from_ref_time(73_056_000 as u64)
```

Comment:
Please rebase onto https://github.com/paritytech/polkadot/pull/7084 once it merged and re-benchmark.